### DISCUSSION TITLE
Restrict Room Creation to a known list of email addresses

### DISCUSSION THREAD

**Discussion by github-user-77** [mozilla/hubs#3275] [Hubs Cloud Questions] [answered]
Hi All, I think I am misunderstanding how accounts in HC work. I want only a known group of people (77 of them) to be able to create rooms, and any person can visit those rooms, but visitors cannot create their own rooms. I thought it meant I should verify the 77 email address as administrators, but am I overcomplicating it? Is there a way in the admin panel to restrict room creation to an approved list of email addresses?

If it is not possible to prevent room creation, then next best option would be prevent anyone other than the 77 from being able to sign-in (i.e., via magic link).

**Comment by robinkwilson** [comment, marked as answer]
I'm afraid we don't have that built as a configuration yet --  Though please make a feature request issue in https://github.com/mozilla/hubs-cloud/issues !
 
You could use a custom client, that hides the create room button if they do not have an account + disable account creation for the visitors. To limit the making of rooms on your instance.

**Reply by github-user-77** [reply]
Thanks @robin-k-wilson for the reply, most appreciated. Will create feature request issue as suggested.
